linux_os/guide/system/accounts/accounts-pam/display_login_attempts/rule.yml

```diff
@@ -9,20 +9,17 @@ documentation_complete: true
 title: 'Ensure PAM Displays Last Logon/Access Notification'
 
 description: |-
-    To configure the system to notify users of last logon/access
-    using <tt>pam_lastlog</tt>, add or correct the <tt>pam_lastlog</tt>
-    settings in
-    <tt>{{{ pam_lastlog_path }}}</tt> to read as follows:
-    <pre>session     required pam_lastlog.so showfailed</pre>
-    And make sure that the <tt>silent</tt> option is not set for
-    <tt>pam_lastlog</tt> module.
+    To configure the system to notify users of last logon/access using <tt>pam_lastlog</tt>,
+    add or correct the <tt>pam_lastlog</tt> settings in <tt>{{{ pam_lastlog_path }}}</tt>
+    to read as follows:
+    <pre>session     [default=1]    pam_lastlog.so nowtmp showfailed</pre>
```

Comment:
for now the Ubuntu 20.04 STIG benchmark still mentions:
`session required pam_lastlog.so showfailed`

Reply:
The STIGs for OL also have the same config entry:
```
session required pam_lastlog.so showfailed
```

Reading the [man pages](https://www.man7.org/linux/man-pages/man5/pam.conf.5.html) there is an equivalent control using the `[...]` syntax:
```
Each of the four keywords: required; requisite; sufficient; and
optional, have an equivalent expression in terms of the [...]
syntax. They are as follows:

       required
           [success=ok new_authtok_reqd=ok ignore=ignore default=bad]

       requisite
           [success=ok new_authtok_reqd=ok ignore=ignore default=die]

       sufficient
           [success=done new_authtok_reqd=done default=ignore]

       optional
           [success=ok new_authtok_reqd=ok default=ignore]
```

Reply:
@freddieRv , we use these two syntaxes in some PAM rules, but in the context of `pam_lastlog` the control is not so relevant to achieve the expected outcome. Many combinations would work fine. In any case, in my last commits I updated the verbiage in rule descriptions to be more flexible. This is similar to my comment in https://github.com/ComplianceAsCode/content/pull/10979#discussion_r1307627997.